src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1085,7 +1084,7 @@ public sealed override bool HasUnsupportedMetadata
 
         #endregion
 
-        internal virtual bool IsIterator
+        public virtual bool IsIterator
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

I see no obvious reason to make this change.

Reply:
The idea was that here `internal` and `public` are identical in behavior and making this `public` can hint, that this property is exposed to the external consumers when working on the compiler codebase. If you don't consider that valuable I'll revert it

Reply:
> If you don't consider that valuable I'll revert it

I consider this change unnecessary. Please revert changes to this file.